MediaRecorder.html

```diff
@@ -272,23 +272,25 @@
 </section>
 
 <section id="blob-event">
-        <h2>Blob Event</h2>
-
-        <dl title='[Constructor] interface BlobEvent : Event' class='idl'>
-  <dt>readonly attribute Blob data</dt>
-  <dd>
-    Returns a Blob object whose type attribute indicates the encoding of the blob data.
-  </dd>
-</dl>
-
-
-        <h3>BlobEventInit</h3>
-  <dl title='dictionary BlobEventInit' class='idl'>
-          <dt>Blob data</dt>
-          <dd>
-            A Blob object containing the data to deliver via this event.
-          </dd>
-        </dl>
+    <h2>Blob Event</h2>
+    <dl title='interface BlobEvent : Event' class='idl'>
+        <dt>Constructor(DOMString type, optional BlobEventInit eventInit)</dt>
```

Comment:
Because data attribute on Blob is not nullable, there cannot be a way to create a BlobEvent without providing a Blob. In other words, this eventInitDict (this is what it's usually called) cannot be optional, and the data member of BlobEventInit must be required.